assets/scss/_universal-login.scss

```diff
@@ -35,6 +28,8 @@ body {
     letter-spacing: -0.100741px;
     color: #252525;
     user-select: none;
+    font-family: 'Inter', sans-serif;
+
```

Comment:
redundant line

Reply:
@VitaliiShpital haven't been able to figure out why this font doesn't get applied here if this line is not here